packages/cli/templates/react/igr-ts/projects/_base/files/package.json

```diff
@@ -8,9 +8,6 @@
     "@testing-library/react": "^14.0.0",
     "element-internals-polyfill": "^1.3.10",
     "functions-have-names": "^1.2.3",
-    "igniteui-dockmanager": "^1.13.0",
-    "igniteui-react": "18.3.0",
-    "igniteui-react-core": "18.3.0",
```

Comment:
if we are removing the core here, we should add it then to all the project which are using it, e.g. pie-chart, grid basic, category chart and so on.
_Note: after creating a react project with the CLI, added some components, and then tried to install manually igniteui-react-core@18.6.0 it failed with `Cannot read properties of undefined (reading 'fileExists')`!_

Reply:
That error is likely from the CLI packages (check the full stack), not really related to the react-core package.
The rest is true though, if that package is a dependency of some (or all) components, views using them should have it listed as a dependency to install.